sympy/printing/latex.py

```diff
@@ -2058,6 +2058,10 @@ def _print_primeomega(self, expr, exp=None):
                     self._print(exp))
         return r'\Omega\left(%s\right)' % self._print(expr.args[0])
 
+    def _print_Quantity(self, expr):
+        if str(expr) == 'degree':
+            return r"^\circ"
+        return r"%s" %expr
```

Comment:
No need for a test case for this line.
Just add a space between `%` and `expr`.